packages/pigeon/pubspec.yaml

```diff
@@ -2,14 +2,14 @@ name: pigeon
 description: Code generator tool to make communication between Flutter and the host platform type-safe and easier.
 repository: https://github.com/flutter/packages/tree/main/packages/pigeon
 issue_tracker: https://github.com/flutter/flutter/issues?q=is%3Aissue+is%3Aopen+label%3Apigeon
-version: 7.0.4 # This must match the version in lib/generator_tools.dart
+version: 7.0.5 # This must match the version in lib/generator_tools.dart
 
 environment:
   sdk: ">=2.12.0 <3.0.0"
   flutter: ">=3.0.0"
 
 dependencies:
-  analyzer: ">=4.4.0 <5.0.0"
+  analyzer: ">=5.0.0 <6.0.0"
```

Comment:
This can become `^5.0.0`. It was only a range because it previously supported a multi-major-version range, and apparently nobody updated that when the lower bound was moved up.